### ISSUE TITLE
Add `Just` library utils from @angus-c

### ISSUE THREAD

**Issue by davidhund** [davidhund/awesome-vanilla-js#10]
I just (re)discovered https://github.com/angus-c/just

It looks like a very valuable VanillaJS util-library (like lodash etc) to add...

**Comment by davidhund** [comment]
@dkniffin What do you think, should we add _libraries_ such as [Just](https://github.com/angus-c/just) or [AlpineJS](https://github.com/alpinejs/alpine) (and _many_ others) or keep this focussed more on _util_ scripts?

**Comment by dkniffin** [comment]
@davidhund 🤔  I think these could be useful to include, though we probably should draw a line somewhere, because a lot of modules on npm have zero dependencies, which means they could potentially be included here.

**Comment by davidhund** [comment]
True. My thought was that explicitely 'Vanilla' libraries could be helpful. Stand-alone modules are probably overkill
